### DISCUSSION TITLE
What 🍎 PC preceded the Mac?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1104] [Galaxy Trivia] [answered]
**Bonus:** Pin 📌 on a map of Interpol headquarters.

**Comment by peterbaccaro** [comment]
The **Apple Lisa** preceded the Macintosh as Apple's personal computer. The Apple Lisa was introduced in 1983, while the Macintosh was introduced in 1984. The Lisa was one of the first personal computers to feature a graphical user interface (GUI) and a mouse, making it more user-friendly and accessible. However, it was relatively expensive and did not achieve widespread success in the market. The Macintosh, with its iconic "1984" Super Bowl commercial, went on to become a more influential and successful product for Apple, shaping the direction of personal computing and popularizing the use of GUIs in personal computers.

The Interpol Headquarters is located at 200 Quai Charles de Gaulle, 69006 Lyon, France.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/82c1e59b-3205-4c10-9a58-652ba1bd4519)

**Comment by thuongtruong1009** [comment, marked as answer]
The Apple II was the personal computer that preceded the Macintosh (Mac). It was introduced by Apple Inc. in 1977 and is considered one of the earliest successful mass-produced microcomputer products.

As for Interpol headquarters, it's located in Lyon, France. However, I'm a text-based AI and I can't directly interact with maps or provide real-time location data. Please use a mapping service to find the exact location of the Interpol headquarters and place a pin on the map yourself.
